recipes/default.rb

```diff
@@ -0,0 +1,36 @@
+include_recipe 'apt'
+
+type = case node['platform_family']
+       when 'fedora', 'rhel'
+         'rpm'
+       else
+         'deb'
+       end
+
+packagecloud_repo 'StackStorm/stable' do
+  type type
+end
+
+package 'st2mistral'
+
+# ensure directories are managed
+[
+  node['openstack-mistral']['etc_dir']
+].each do |path|
+  directory "creating mistral directory #{path}" do
```

Comment:
I think the directory resource should be left, but hard coded here.  Then the owner, group, mode verified that they are the same as the packaging.  This allows you to have chef make sure the permissions on the dir don't change.  Thoughts?